---
identifier: "CVE-2022-34529"
identifiers:
- "GHSA-gq4p-4hxv-5rg9"
- "CVE-2022-34529"
package_slug: "pypi/pywasm3"
title: "Out-of-bounds Read"
description: "WASM3 v0.5.0 was discovered to contain a segmentation fault via the
  component Compile_Memory_CopyFill."
date: "2023-08-18"
pubdate: "2022-07-28"
affected_range: "<=0.5.0"
fixed_versions: []
affected_versions: "All versions up to 0.5.0"
not_impacted: ""
solution: "Unfortunately, there is no solution available yet."
urls:
- "https://nvd.nist.gov/vuln/detail/CVE-2022-34529"
- "https://github.com/wasm3/wasm3/issues/337"
- "https://github.com/pypa/advisory-database/tree/main/vulns/pywasm3/PYSEC-2022-43057.yaml"
- "https://github.com/advisories/GHSA-gq4p-4hxv-5rg9"
cvss_v3: "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
uuid: "b5fdb86c-0f11-4cb6-9702-df4862fd0df8"
cwe_ids:
- "CWE-1035"
- "CWE-125"
- "CWE-937"
